app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
Ok. To be honest I don't really know what you think needs to be changed anymore :D I thought that you wanted to remove the `.directive` from the file name and I couldn't really find the exact moment you changed your mind. I can't really comment on every aspect you take and I'm not really sure you would like to hear my thoughts on the wordings I use. You've mentioned quite a few details I haven't touched, but rather skimmed as irrelevant, so I might have missed some things too.

Maybe you could sum the change request?